src/components/MapExplorer.jsx

```diff
@@ -1,12 +1,113 @@
+  GOOGLE_MAPS_API_KEY: "AIzaSyAzMva99KALzUQfe_BCkCovZyGK-dld08s",
+
+  googleMap: undefined,
+  
+  getInitialState: function() {
+    this.googleMap = undefined;    
+    return {
+    };
+  },
+  
   render() {
+    window.initMapExplorer = this.initMapExplorer;
+    
     return (
       <div className="map-explorer">
-        <h2>Map Explorer</h2>
-        <p>THIS PAGE SHOULD EXPAND TO FIT THE WINDOW ...minus any header/footer.</p>
+        <div ref="mapVisuals" className="map-visuals"></div>
+        <div className="map-controls">
+          <Script src={'https://maps.googleapis.com/maps/api/js?key='+this.GOOGLE_MAPS_API_KEY+'&callback=initMapExplorer'}>{
+            ({done}) => !done ? <div>Google Maps API is loading</div> : <div>Google Maps is ready</div>
+          }</Script>
+        </div>
       </div>
     );
+  },
+
+  //Once React component has rendered, process the map.
+  //See notes on initMapExlorer().
+  componentDidMount() {
+    if (window.google && window.google.maps) {
+      console.log("componentDidMount: window.google found.");
+      this.initMapExplorer();
+    } else {
+      console.log("componentDidMount: window.google not found.");
+    }
+  },
+  
+  //Cleanup!
+  componentWillUnmount() {
+    //Note that the Map Explorer (and hence, the Google Map) is recreated every
+    //time the Map page is opened/navigated to. As such, we want to make sure we
+    //wipe all references/handles to prevent a memory leak.
+    this.googleMap = undefined;
+  },
+  
+  //Initialises the Map Explorer.
+  //NOTE: initMapExlorer() can be called in two ways:
+  //1. When the map is loaded for the first time, the <Script> for the Google
+  //   Maps API needs to be loaded dynamically. When it's successfully loaded,
+  //   initMapExplorer is called back.
+  //2. If the <Script> has been loaded previously - e.g. the user navigated
+  //   away from the page and has now returned - then initMapExplorer() will be
+  //   called during componentDidMount(), after the necessary HTML elements have
+  //   been rendered.
+  initMapExplorer() {
+    console.log("MapExplorer.initMapExplorer()");
+    
+    this.googleMap = new google.maps.Map(this.refs.mapVisuals, {
+      center: { lat: Cameras.median.lat, lng: Cameras.median.lng },
+      zoom: 11
+    });
+    
+    this.paintAllCameras();
+  },
+  
+  //PLACEHOLDER - shaun.a.noordin@zooniverse.org to do something about this.
+  //Like, make it a proper function with user-selected filters.
+  paintAllCameras() {
+    console.log("MapExplorer.paintAllCameras()");
+    
+    //Sanity Check
+    if (!(window.google && window.google.maps && this.googleMap)) {
+      console.log("ERROR: MapExplorer.paintAllCameras() doesn't have a map to paint on!");
+      return;  
+    }
+    
+    var inputColour = "#f39";
+    var inputOpacity = 0.5;
+    var inputSize = 500;
+    
+    for (var i = 0, camera; camera = Cameras.all[i]; i++) {
+      var marker = new google.maps.Circle({
```

Comment:
That's a good point, @rogerhutchings - for the moment, I only see the markers as simple circles, but I do plan on expanding them with, e.g. animation, for future phases. I'll need to experiment with that first though, so for now I'm leaving the marker objects + the Data Overlay/painting feature rolled up in MapExplorer.

(On that note, I may need to update my Features chart with an indicator of which feature goes into which component - https://github.com/zooniverse/wildcam-gorongosa-education/issues/1#issuecomment-171797240 )